packages/reactivity/__tests__/effectScope.spec.ts

```diff
@@ -192,6 +192,25 @@ describe('reactivity/effect/scope', () => {
     expect(dummy).toBe(7)
   })
 
+  it('should warn onDispose() is called when there is no active effect scope', () => {
+    const spy = jest.fn()
+    const scope = new EffectScope()
+    scope.run(() => {
+      onScopeDispose(spy)
+    })
+
+    expect(spy).toHaveBeenCalledTimes(0)
+
+    onScopeDispose(spy)
+
+    expect(
+      '[Vue warn] onDispose() is called when there is no active effect scope  to be associated with.'
```

Comment:
There seems to be an extra space here to remove in the actual warning message (and in the test)
```suggestion
      '[Vue warn] onDispose() is called when there is no active effect scope to be associated with.'
```

Reply:
has to fix! thanks~

Reply:
has to fix~ @posva